src/Gamepad/I18n.elm

```diff
@@ -0,0 +1,62 @@
+module Gamepad.I18n exposing (Lang(..), Translations, getLocale)
+
+
+type alias Translations =
+    { noGamepadsDetected : String
+    , remappingGamepadComplete : Int -> String
+    , pressAnyButtonToGoBack : String
+    , remappingGamepad : Int -> String
+    , press : String
```

Comment:
This could become a `String -> String` function, too, the first argument being the name of the button.

Reply:
It would mean restructuring the html because currently the name of the button is shown in a different div. We could also add a `buttonName : Digital -> String` to provide default translations.

Reply:
You are right, but I it shouldn't be too difficult to change the html.

Unfortunately the library confuses button names (X, A, DpadUp, Left Trigger...) and action names (Jump, Fire, Talk, Dash...): button names are not really important when remapping, and action names are provided already by the user code, so they can be translated as the programmer prefers.